### ISSUE TITLE
Add unit tests for Merge Sort and Quick Sort in Java

### ISSUE THREAD

**Issue by larissalages** [larissalages/code_problems#60]
We have the code for Merge Sort and Quick Sort in java but we don't have unit tests for them, please create a separated file for test the class of one of these algorithms.

**Comment by Tripathi-Alok** [comment]
Can i work on this?

**Comment by sumitsawant** [comment]
I can also work on this .

**Comment by sumitsawant** [comment]
Maybe @Tripathi-Alok can work on one algorithm and I can work on one?

**Comment by sumitsawant** [comment]
@larissalages what do you suggest?

**Comment by Tripathi-Alok** [comment]
Yes i can work on merge sort and @sumitsawant  may work on Quick Sort.
what you say @larissalages

**Comment by larissalages** [comment]
Sure @sumitsawant you can do that! choose an algorithm to work on, the other person can work on the other, thanks!

**Comment by sumitsawant** [comment]
I'll work on @larissalages Merge sort . Please assign me that!

**Comment by Diegoslourenco** [comment]
@larissalages Can I help with this one?
